src/GraphQL.Tests/Types/UShortGraphTypeTests.cs

```diff
@@ -15,25 +15,25 @@ public void Coerces_given_inputs_to_null()
         [InlineData(-999999)]
         [InlineData(-1)]
         [InlineData(999999)]
-        [InlineData("999999")]
-        [InlineData("-999999")]
-        [InlineData("-1")]
         public void Coerces_given_inputs_to_overflow_exception(object input) =>
             AssertException<OverflowException>(input);
 
         [Theory]
+        [InlineData("999999")]
+        [InlineData("-999999")]
+        [InlineData("-1")]
         [InlineData("abc")]
         [InlineData("-999999.99")]
+        [InlineData("123")]
+        [InlineData("65535")]
         public void Coerces_given_inputs_to_out_of_bound_exception(object input) =>
-            AssertException<FormatException>(input);
+            AssertException<ArgumentException>(input);
 
         [Fact]
         public void Coerces_double_to_invalid_operation_exception()
-            => AssertException<InvalidOperationException>(999999.99);
+            => AssertException<ArgumentException>(999999.99);
```

Comment:
All such tests can be merged with the previous test if the same type of exception is thrown.